code/datums/gamemode/dynamic/dynamic_rulesets.dm

```diff
@@ -353,8 +353,11 @@
 	return ..()
 
 /datum/dynamic_ruleset/proc/latejoinprompt(var/mob/user, var/ruleset)
```

Comment:
`ruleset` seems useless, it's always equal to `src`. Since you're cleaning this up you can consider getting rid of it

Reply:
Done